ecosystem/sep-0006.md

```diff
@@ -355,7 +357,9 @@ Example:
 
 #### Transaction Polling
 
-Once the deposit or withdraw request has been made and KYC information has been passed to the anchor via [SEP-12](sep-0012.md), the wallet will need to detect that the anchor has completed the flow by sending funds to the user. The wallet needs to poll the [Transaction History](#transaction-history) endpoint response from the anchor and match the transaction objects returned using the `memo` and `memo_type` parameters used in the deposit or withdraw success response.
+Once the deposit or withdraw request has been made and KYC information has been passed to the anchor via [SEP-12](sep-0012.md), the wallet will need to detect that the anchor has completed the flow by sending funds to the user. 
+
+The wallet either needs to provide the anchor with an `on_change_callback` or to poll the [Transaction History](#transaction-history) endpoint response from the anchor and match the transaction objects returned using the `memo` and `memo_type` parameters used in the deposit or withdraw success response.
```

Comment:
❔  Is there no way for the client to look at the specific transaction and its status?

Reply:
There is. The client can pass the `on_change_callback` and get updates on that specific transaction, or the client can poll `GET /transaction?id=`.  I think you confusion is understandable though, because we mention the `GET /transactions` (plural) endpoint. Instead of `GET /transaction?id=`.

Reply:
Oh I see why. Its because the success response for `/deposit` and `/withdraw` do not include the transaction ID. So there's no way for the client to query the transaction using `GET /transaction?id=` until it matches the `memo`

Reply:
I'll make a separate PR to add the `id` attribute to the response. I don't know why it isn't there right now. It really should be required in the response, but I'll have to make it optional in order to be non-breaking.

Reply:
stellar/stellar-protocol#860